prow/cmd/hook/main.go

```diff
@@ -171,5 +183,23 @@ func main() {
 	// Serve plugin help information from /plugin-help.
 	http.Handle("/plugin-help", pluginhelp.NewHelpAgent(pluginAgent, githubClient))
 
+	var appClient config.Client
+	if err := yaml.Unmarshal(bytes.TrimSpace(gitAppClientRaw), &appClient); err != nil {
+		logger.WithError(err).Fatal("Error unmarshalling git app client config.")
+	}
+	cookie := sessions.NewCookieStore(securecookie.GenerateRandomKey(64))
```

Comment:
The cookie store key needs to be constant between instances. We should have an extra secret, or have it store it in some Kubernetes API object (but I don't think we have that functionality currently).

Reply:
[+1](https://github.com/kubernetes/test-infra/pull/6629#discussion_r166092012), we should have a secret for this.

Reply:
done